src/features/game/expansion/components/resources/Shrub.tsx

```diff
@@ -0,0 +1,255 @@
+        content: `+${shrub.wood.amount}`,
+      });
+
+      await new Promise((res) => setTimeout(res, 2000));
+      setCollecting(false);
+    } catch (e: any) {
+      displayPopover(
+        <span className="text-xs text-white text-shadow">{e.message}</span>
+      );
+    }
+  };
+
+  const timeLeft = getTimeLeft(shrub.wood.choppedAt, SHRUB_RECOVERY_TIME);
+
+  return (
+    <div
+      className="relative z-10 w-full h-full"
+      onMouseEnter={handleMouseHoverShrub}
+      onMouseLeave={handleMouseLeaveShrub}
+    >
+      {!chopped && (
+        <div
+          ref={containerRef}
+          className="group cursor-pointer w-full h-full"
+          onClick={strike}
+        >
+          <Spritesheet
+            className="group-hover:img-highlight pointer-events-none z-10"
+            style={{
+              position: "absolute",
+              left: "-24px",
+              top: "-2px",
+            }}
+            getInstance={(spritesheet) => {
+              sparkGif.current = spritesheet;
+            }}
+            image={sparkSheet}
+            widthFrame={96}
+            heightFrame={64}
+            fps={24}
+            steps={7}
+            direction={`forward`}
+            autoplay={false}
+            loop={false}
+            onLoopComplete={(spritesheet) => {
+              spritesheet.pause();
+            }}
+          />
+        </div>
+      )}
+
+      <Spritesheet
+        style={{
+          position: "absolute",
+          left: "-74px",
+          top: "-14px",
+          opacity: collecting ? 1 : 0,
+          transition: "opacity 0.2s ease-in",
+          imageRendering: "pixelated",
+        }}
+        className="pointer-events-none z-20"
+        getInstance={(spritesheet) => {
+          choppedGif.current = spritesheet;
+        }}
+        image={dropSheet}
+        widthFrame={133}
+        heightFrame={84}
+        fps={20}
+        steps={11}
+        direction={`forward`}
+        autoplay={false}
+        loop={false}
+        onLoopComplete={(spritesheet) => {
+          spritesheet.pause();
+        }}
+      />
+
+      {chopped && (
+        <div className="absolute" style={{ top: "12px", left: "8px" }}>
+          <img
+            src={smallShrub}
+            className="pointer-events-none -z-10 opacity-50"
+          />
+        </div>
+      )}
+
+      <div
+        className={classNames(
+          "absolute top-16 left-5 transition-opacity pointer-events-none content-center",
+          {
+            "opacity-100": touchCount > 0,
+            "opacity-0": touchCount === 0,
+          }
+        )}
+      >
+        <HealthBar percentage={collecting ? 0 : 100 - (touchCount / 2) * 100} />
+      </div>
+
+      {true && (
+        <div
+          className="absolute"
+          id="chopped-shrub"
+          style={{
+            bottom: "60px",
+            left: "-25px",
+          }}
+        >
+          <TimeLeftPanel
+            text="Recovers in:"
```

Comment:
<img width="211" alt="Screen Shot 2022-07-08 at 9 38 41 am" src="https://user-images.githubusercontent.com/11745561/177888768-be2bf426-a80a-4ef0-b1e6-562e207a44f1.png">

A bit of a visual bug, the time is missing.

A would suggest removing the `Recovers in` and just showing the time. The modal is a bit clunky with the extra text.